src/core/ui/components/sign-in/index.tsx

```diff
@@ -103,9 +103,14 @@ function SignInRoutes(): JSX.Element {
           loginMethods={[...web3LoginMethods, ...web2LoginMethods]}
         >
           <Flow.Root flow="sign-in">
-            <Route index>
-              <SignInStart />
-            </Route>
+            <Switch>
+              <Route path="all-wallets">
+                <SignInStart showAllWallets showBackButton />
+              </Route>
```

Comment:
I guess we are using this one for the theme page right?

Reply:
Yeah, this is `viewOnly`